doc/devel/development_setup.rst

```diff
@@ -164,6 +164,22 @@ true for ``*.py`` files.  If you change the C-extension source (which might
 also happen if you change branches) you will have to re-run
 ``python -m pip install -ve .``
 
+Verify the Installation
+=======================
+
+Run the following command to make sure you have correctly installed Matplotlib in editable mode. 
+The command should be run when the virtual environment is activated ::
+   
+    python -c "import matplotlib; print(matplotlib.__file__)"
+  
```

Comment:
```suggestion
Run the following command to make sure you have correctly installed Matplotlib in editable mode.
The command should be run when the virtual environment is activated ::

    python -c "import matplotlib; print(matplotlib.__file__)"

```

Hint: selecting text on github makes it easy to find where the trailing whitespace is